st2tests/st2tests/fixtures/packs/BUILD

```diff
@@ -22,6 +22,7 @@ shell_sources(
 
 python_sources(
     name="test_content_version",
+    skip_black=True,
```

Comment:
This is in a git submodule, so we don't want black to try to change anything across the git boundaries.

Reply:
Maybe worth a comment?

Reply:
There's a comment mentioning the git submodule at the top of this file:
https://github.com/StackStorm/st2/blob/3e5d8df808c5843134116c1989ccf20ef3637ff8/st2tests/st2tests/fixtures/packs/BUILD#L1-L2

I suppose we can extend that comment if needed, because there are 3 targets that cover that git submodule. The shell_sources target has `skip_shellcheck=True` for similar reasons: don't complain about things across the git boundaries.

Reply:
Can have a comment here to explain this one off disable

Reply:
Oops. I missed this before merging. So, I added a comment in #5777: https://github.com/StackStorm/st2/pull/5777/commits/f443f391a3a3f800a31babb0db44a7303bd2f52b